lib/mongorito.js

```diff
@@ -56,21 +71,23 @@ class Mongorito {
         options = arg;
       }
     });
-
-    // convert mongo:// urls to monk-supported ones
     urls = urls.map(function (url) {
-      return url.replace(/^mongo\:\/\//, '');
+      if(!url.startsWith('mongodb://')){
+        url = 'mongodb://' + url;
+      }
+      return url;
     });
 
-    let db = monk(urls, options);
-
-    // if there is already a connection
-    // don't overwrite it with a new one
-    if (!this.db) {
-      this.db = db;
-    }
-
-    return db;
+    return new Promise(function(resolve, reject){
```

Comment:
This is wrapped so that Ic an handle the instance setup then still return a promise to make connect yieldable without forcing it to be used in a co wrapper. Without doing this and instead using yields old connect strings will break. If backward compability is not an issue this function could change to 

```js
  static * connect () {
    // parse arguments
    let args = Array.prototype.slice.call(arguments);

    let urls = [];
    let options = {};

    args.forEach(function (arg) {
      if (is.string(arg)) {
        urls.push(arg);
      }

      if (is.object(arg)) {
        options = arg;
      }
    });
    urls = urls.map(function (url) {
      if(!url.startsWith('mongodb://')){
        url = 'mongodb://' + url;
      }
      return url;
    });

    let db = yield MongoClient.connect(urls[0], options);
    if(!Mongorito.instance.db) {
      Mongorito.instance.db = db;
    }
    return db;
  }
```